### ISSUE TITLE
Empty function calls have unwanted padding

### ISSUE THREAD

**Issue by chapmandu** [jcberquist/commandbox-cfformat#36]
Input:
`return $$badRequest();`

Output:
`return $$badRequest( );`

Expected:
`return $$badRequest();`

Settings:
```
{
	"array.empty_padding": false,
	"array.multiline.element_count": 4,
	"array.multiline.leading_comma": false,
	"array.multiline.leading_comma.padding": true,
	"array.multiline.min_length": 40,
	"array.padding": false,
	"binary_operators.padding": true,
	"brackets.padding": false,
	"for_loop_semicolons.padding": true,
	"function_call.casing.builtin": "pascal",
	"function_call.casing.userdefined": "camel",
	"function_call.empty_padding": false,
	"function_call.multiline.element_count": 4,
	"function_call.multiline.leading_comma": false,
	"function_call.multiline.leading_comma.padding": true,
	"function_call.multiline.min_length": 40,
	"function_call.padding": false,
	"function_declaration.empty_padding": false,
	"function_declaration.group_to_block_spacing": "spaced",
	"function_declaration.multiline.element_count": 4,
	"function_declaration.multiline.leading_comma": false,
	"function_declaration.multiline.leading_comma.padding": true,
	"function_declaration.multiline.min_length": 40,
	"function_declaration.padding": false,
	"indent_size": 2,
	"keywords.block_to_keyword_spacing": "spaced",
	"keywords.empty_group_spacing": false,
	"keywords.group_to_block_spacing": "spaced",
	"keywords.padding_inside_group": false,
	"keywords.spacing_to_block": "spaced",
	"keywords.spacing_to_group": true,
	"max_columns": 120,
	"parentheses.padding": false,
	"strings.quote": "double",
	"strings.attributes.quote": "double",
	"struct.empty_padding": false,
	"struct.multiline.element_count": 4,
	"struct.multiline.leading_comma": false,
	"struct.multiline.leading_comma.padding": true,
	"struct.multiline.min_length": 40,
	"struct.padding": false,
	"struct.separator": " = ",
	"tab_indent": true
}

```

**Comment by jcberquist** [comment]
Thanks! This looks to be an issue with the upstream CFML syntax - it doesn't see and highlight `$$badRequest()` as a function call, so the formatting doesn't apply the function call rules.

**Comment by jcberquist** [comment]
This should be fixed in `v0.12.2`.
